ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallViewListener.kt

```diff
@@ -12,5 +12,5 @@ interface PaywallViewListener {
     fun onRestoreStarted() {}
     fun onRestoreCompleted(customerInfo: CustomerInfo) {}
     fun onRestoreError(error: PurchasesError) {}
-    fun onDismissed() {}
+    fun onCloseButtonPressed() {}
```

Comment:
I'm changing this listener method, since the `onDismissed` only made sense when displaying as a dialog and the dialog options will already have a separate callback for that. 
I'm making showing the close button optional (currently not implemented) and having a callback here for when it's tapped.